src/framework/mpas_stream_manager.F

```diff
@@ -4845,6 +4964,51 @@ logical function is_decomposed_dim(dimName) !{{{
 
     end function is_decomposed_dim !}}}
 
+    
+    
+    integer function global_dim_size(block, dimNames, dimSizes, nDim) !{{{
+    use iso_c_binding, only : c_sizeof
```

Comment:
Let's follow the convention of three or four spaces of indentation for function bodies.

Reply:
Also, it doesn't look like `c_sizeof` is used in this function?